src/main/java/com/goncharenko/currencyexchangerate/domain/Role.java

```diff
@@ -0,0 +1,54 @@
+package com.goncharenko.currencyexchangerate.domain;
+
+import org.springframework.security.core.GrantedAuthority;
+
+import java.util.Objects;
+
+public class Role implements GrantedAuthority {
```

Comment:
Lets use lombok @Getter @Setter instead of java plain code and @EqualsAndHashCode